internal/services/executors/transport/graphql/executor_resolver.go

```diff
@@ -20,6 +20,8 @@ type ExecutorResolver struct {
 	executor types.Executor
 }
 
+const oneMonth = 30 * 24 * time.Hour
```

Comment:
I _think_ the worst case scenario for the gap between releases is 33 days if the 22nd falls on a Sunday in 31 day month, in which case the release would occur on the 20th, and then on the 22nd the following month, per [the release algorithm](﻿https://handbook.sourcegraph.com/departments/engineering/dev/process/releases/#when-we-release).

Also, releases aren't always very consistent in their timing within the day, and we have teammates all over the world so there's theoretically about a 39 hour window (from UTC-11 to UTC+14, which covers all the world's inhabited time zones) in which a release can occur.

Also _also_, there could be a DST transition in there, which could add an extra hour.

(Why yes, I have worked on a standard library date package, why do you ask?)

So, to be extra safe, how about:

```suggestion
const oneMonth = 35 * 24 * time.Hour
```

(with the corresponding comment updates below, which I won't bother suggesting)

Reply:
I love it when you talk about `date/time`. It's amazing how much you know off the top of your head.

And yes, thank you for this suggestion.

Reply:
Just for my understanding, is there a reason the insider executor versions do not have version in the string?

Reply:
@eseliger can provide more context here. 

However, I remember asking a question similar to this about the difference between `productVersion` and `buildVersion`, and Sander explained why we have that in the first place. You can [check it out here](https://sourcegraph.slack.com/archives/C07KZF47K/p1660074146184129?thread_ts=1660072137.189849&cid=C07KZF47K).